dockerfiles/centos_7_1.x

```diff
@@ -83,8 +83,26 @@ RUN set -x \
         ldconfig ) \
     && : "---------- gperftools ----------" \
     && yum install -y gperftools-libs \
-    && (GOPATH=/usr/src/go go get github.com/google/pprof; \
-        cp /usr/src/go/bin/pprof /usr/local/bin) \
+    && : "---------- pprof for gperftools ----------" \
```

Comment:
EPEL repostiory for CentOS 7 offers golang-1.13.11-1.el7. I guess it is not needed here.